app/src/main/java/org/oppia/android/app/splash/SplashActivityPresenter.kt

```diff
@@ -64,12 +67,46 @@ class SplashActivityPresenter @Inject constructor(
     subscribeToOnboardingFlow()
   }
 
+  /** Handles cases where the user clicks the close app option on a deprecation notice dialog. */
   fun handleOnCloseAppButtonClicked() {
     // If the app close button is clicked for the deprecation notice, finish the activity to close
     // the app.
     activity.finish()
   }
 
+  /** Handle cases where the user clicks the update option on a deprecation notice dialog. */
```

Comment:
Nit: ``Handles``. Ditto for the method below.

Reply:
I have modified the instances of "Handle" instead of "Handles".